roles/connectivity_troubleshooter_local/tasks/main.yml

```diff
@@ -86,7 +86,7 @@
       register: connectivity_troubleshooter_local__result_eval_security_groups
 
     - name: Gather information about source subnet network ACLs
-      community.aws.ec2_vpc_nacl_info:
+      amazon.aws.ec2_vpc_nacl_info:
```

Comment:
This module was promoted to `amazon.aws` in the [9.0.0 release](https://github.com/ansible-collections/amazon.aws/blob/main/changelogs/changelog.yaml#L3299).